src/zDAOClient.ts

```diff
@@ -0,0 +1,288 @@
+import {
+  Erc20Transfer as GnosisErc20Transfer,
+  Erc721Transfer as GnosisErc721Transfer,
+  NativeCoinTransfer as GnosisNativeCoinTransfer,
+  Transaction as GnosisTransaction,
+  Transfer as GnosisTransfer,
+  TransferInfo as GnosisTransferInfo,
+} from '@gnosis.pm/safe-react-gateway-sdk';
+import { ethers } from 'ethers';
+
+import GnosisSafeClient from './gnosis-safe';
+import SnapshotClient from './snapshot-io';
+import { SnapshotProposal } from './snapshot-io/types';
+import {
+  AssetType,
+  Config,
+  CreateProposalParams,
+  Proposal,
+  ProposalId,
+  Transaction,
+  TransactionStatus,
+  TransactionType,
+  TransferInfo,
+  VoteChoice,
+  zDAO,
+  zDAOAssets,
+  zDAOId,
+  zNA,
+} from './types';
+import zProposal from './zProposal';
+
+class zDAOClient implements zDAO {
```

Comment:
`zDAOClient.ts`, `zProposal.ts` (which should be renamed to `ProposalClient.ts`, and `SDKClient.ts` (remove the z since it's not accurate) should all be moved into their own folder `clients/` and exported from there. The root folder of the SDK has become too bloated.